lib/ansible/modules/packaging/os/yum.py

```diff
@@ -33,78 +33,87 @@
         several packages this can accept a comma separated list of packages
         or (as of 2.0) a list of packages.
     required: true
-    aliases: [ pkg ]
+    aliases: ['pkg']
   exclude:
     description:
-      - "Package name(s) to exclude when state=present, or latest"
```

Comment:
There's a lot of churn on changing double to single quotes, I could understand wanting to be consistent but it seems outside the scope of this change.